spaceduck/src/Views/RegisterView.jsx

```diff
@@ -1,22 +1,93 @@
 import React from 'react';
 import Header from '../components/Header.jsx'
-
+import './forms-style.css'
+import nyanDuck from '../assets/Nyan_kaczka.png'
 class Register extends React.Component {
-    constructor(props) {
-        super(props);
-       
+    constructor() {
+        super();
+        this.state = {
+            name:'',
+            mail: '',
+            password:'',
+            password2:'',
```

Comment:
Wiem, że to jest czepanie się, ale nie lepiej mieć cos w stylu repetedPassword?
Bo to wygląda trochę, jakby ktoś miał mieć dwa hasła.